views/site-index.jsx

```diff
@@ -10,5 +10,11 @@ export function SiteIndex({ title, _layout }) {
 		<h1>{title}</h1>
 
 		{content}
+
+		<h2>Also available:</h2>
+		<ul>
+			<li><a href="/?fragment=yes">fragment-only</a></li>
```

Comment:
I remain undecided here: On the one hand, it's good to illustrate the possibilities, but on the other hand, `?fragment` is really not something I'd want to encourage (i.e. [this line](https://github.com/complate/complate-sample-express/blob/af5629a1bb66da3966d3398f41ef52654a74e8da/src/index.js#L15) was never meant to be permanent).